src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue

```diff
@@ -123,9 +123,9 @@ async function updateUI() {
         instance.errorCount = failedMessageStore.errorCount;
         instance.isScMonitoringDisconnected = false;
       }
-      endpoint.value.isStale = endpoint.value.isStale && instance.isStale;
       negativeCriticalTimeIsPresent.value |= formatGraphDuration(instance.metrics.criticalTime).value < 0;
```

Comment:
it won't cause flickering, since it doesn't change the rendering hierarchy like the isStale did, but it will potentially cause intermediate values to show. Will extract